rtv/subreddit.py

```diff
@@ -73,6 +74,24 @@ def prompt_subreddit(self):
         if name is not None:
             self.refresh_content(order='ignore', name=name)
 
+    @SubredditController.register(Command('SUBREDDIT_FRONTPAGE'))
+    def show_frontpage(self):
```

Comment:
I'm trying to add a unit test, but I'm getting a VCR error.
vcr.errors.CannotOverwriteExistingCassetteException: No match for the request (<Request (GET) https://api.reddit.com/r/aww/.json?limit=1024>) was found. Can't overwrite existing cassette

In the README, you specified that I should be generating a refresh token so that VCR can change the read-only status of the cassettes to allow writing, but ,y ~/.config/rtv/refresh_token file doesn't exist. I've logged into the client before, and my config file has the persistent=True and clear_auth=False.

Am I supposed to create the refresh_token file manually? Whenever I log in to my account with the client, I can see the refresh_token file in my files. But it very quickly disappears, as if it is a temporary file or something. Am I missing something?